reference/docs-conceptual/components/console/Using-Tab-Expansion.md

```diff
@@ -6,22 +6,35 @@ ms.assetid:  c8730471-bf6a-43b8-ab1d-f9ef5a74f04e
 ---
 # Using Tab Expansion
 
-Command-line shells often provide a way to complete the names of long files or commands automatically, speeding up command entry and providing . Windows PowerShell allows you to fill in file names and cmdlet names by pressing the **Tab** key.
+Command-line shells often provide a way to complete the names of long files or commands
+automatically, speeding up command entry and providing . Windows PowerShell allows you to fill in
```

Comment:
Two changes:
- Looks like a missing word before the period.
- Remove 'Windows'.

Reply:
Found and fixed